scripts/github_actions/generate_build_matrix.py

```diff
@@ -272,7 +272,7 @@ def generate_build_matrix(
                 else ["11.8.0", "12.1.0", "12.4.0"]
             ),
         },
-        "2.5.0": {
+        "2.5.1": {
```

Comment:
Could you create a new entry for `2.5.1` and keep the original `2.5.0`?

Reply:
done